connect/runtime/src/test/java/org/apache/kafka/connect/runtime/standalone/StandaloneHerderTest.java

```diff
@@ -223,93 +216,81 @@ public void testCreateConnectorAlreadyExists() throws Throwable {
         // Second should fail
         FutureCallback<Herder.Created<ConnectorInfo>> failedCreateCallback = new FutureCallback<>();
         herder.putConnectorConfig(CONNECTOR_NAME, config, false, failedCreateCallback);
+        verify(loaderSwap, times(2)).close();
```

Comment:
Can we move this to the end of the test, or at least after invoking `failedCreateCallback::get`? Otherwise the test is racy since we're not guaranteed that `loaderSwap::close` has actually been invoked twice at this point.

Reply:
Done and pushed.